1_fetch/src/fetch_sb.R

```diff
@@ -0,0 +1,10 @@
+# Get the data from ScienceBase
+fetch_sb <- function(project_output_dir = '1_fetch/out') {
+  if(!dir.exists(project_output_dir)){
+    dir.create(project_output_dir)
+  }
+  sb_fp <- file.path(project_output_dir, 'model_RMSEs.csv')
```

Comment:
In  terms of reproducibility, it would be better to have the configurable parts passed in as arguments, so the item hash, the remote filename, and the output filename all passed it. It would make it so that this function could easily be reused for other sbitem downloads. Something potentially `sb_file = "me_RMSE.csv"`, `out_file = "1_fetch/out/model_RMSEs.csv"`, and `sbitem = "5d925066e4b0c4f70d0d0599"`.

Reply:
change made